src/Engine/ProtoCore/FFI/FFIExecutionManager.cs

```diff
@@ -15,7 +15,7 @@ private FFIExecutionManager()
         }
 
         Dictionary<RuntimeCore, FFIExecutionSession> mSessions = new Dictionary<RuntimeCore, FFIExecutionSession>();
-        ExtensionAppLoader mApploader = new ExtensionAppLoader();
+        ExtensionAppLoader mApploader = ExtensionAppLoader.Instance;
```

Comment:
FFIEXecutionManager is singleton, so how are we creating multiple instances of ExtensionAppLoader?